app/components/baseComponents/models/tableData.tsx

```diff
@@ -10,7 +10,7 @@ export interface TableFooter {
 
 export interface TableData {
     caption?: string, 
-    columnNames: Array<String>,
+    columnNames: Array<string>,
```

Comment:
💟 TIL: [this is why](https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string)

Reply:
Ooooo